programs/nft-voter/tests/relinquish_nft_vote.rs

```diff
@@ -454,3 +455,90 @@ async fn test_relinquish_nft_vote_with_unexpired_vote_weight_record() -> Result<
 
     Ok(())
 }
+
```

Comment:
What do you think about adding a test to make sure that you can relinquish as the delegate?